scripts/ensure-ecr-repository.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+ensure_repository() {
+    local aws_service=$1
+    local ecr_repo=$2
+    local region="us-east-1"
+    local ecr_tpl_file_path=""
+
+    if (echo "$ecr_repo" | grep -q "controller"); then
+        ecr_tpl_file_path="$THIS_DIR/templates/ecr_controller_template.txt"
+    else
+        ecr_tpl_file_path="$THIS_DIR/templates/ecr_chart_template.txt"
+    fi
+
+    export aws_service
+    local catalog_data=$(envsubst < $ecr_tpl_file_path 2>&1)
```

Comment:
clever, but we need to verify that this tool is available inside the image that we use to run ACK CD jobs.